### ISSUE TITLE
Cron, on Sunday

### ISSUE THREAD

**Issue by ebrucki** [bunkat/later#152]
cron('0 0 * * 6,7') should be on Saturday and Sunday.
1-6 works for Monday to Saturday. It writes 2-7 to .d, but cron 7, which is Sunday, adds 8 to .d, instead of 1.

**Comment by bunkat** [comment]
Does 0, 6 do what you want?  Or SAT, SUN?  Or are they all broken at the moment?

**Comment by ebrucki** [comment]
Yes, everything works, no problem. It just caused me a lot of frustration as, according to Wikipedia, 7 should also work.

**Comment by pekeler** [comment]
Yea, this is broken. Weekdays in Cron are

0 SUN
1 MON
2 TUE
3 WED
4 THU
5 FRI
6 SAT
7 SUN

Later's internal representation of weekdays are:

1 SUN
2 MON
3 TUE
4 WED
5 THU
6 FRI
7 SAT

So far so good. The test title https://github.com/bunkat/later/blob/5d38cc3779ad0ee42d61675c8209c1e47c07ca93/test/parse/cron-test.js#L311 'should parse 7 as Sunday' is correct, but the implementation is incorrect. It should be

```JavaScript
		it('should parse 7 as Sunday', function() {
			var p = parse('* * * * * 7', true);
			p.schedules[0].should.eql({d: [0]});
		});
```

Right now, Later parses '* * * * * 7' as Saturday which is incorrect.
